roles/ara_api/tasks/install/pypi.yaml

```
@@ -2,9 +2,11 @@
 - name: Install required packages from Pypi
   pip:
     name: ara[server]
-    state: present
+    state: "{% if ara_api_version == 'latest' %}latest{% else %}present{% endif %}"
+    version: "{% if ara_api_version != 'latest' %}{{ ara_api_version }}{% else %}{{ omit }}{% endif %}"
```

More of a style nitpick because your approach works but for simple one-liner if/else's we can use ternaries like elsewhere in the role:
```suggestion
    state: ""{{ (ara_api_version == 'latest') | ternary('latest', 'present') }}""
    version: ""{{ (ara_api_version != 'latest') | ternary(ara_api_version, omit) }}""
```

---

That's a really nice filter, never really used it. But it's a lot cleaner then inline jinja ;-) Thanks for the tip! (now if you'll excuse me, I need to fix a lot of other roles :joy: )